Coman C‑673_16

```diff
@@ -331,7 +331,7 @@ In that regard, as is apparent from the Explanations relating to the Charter of
 It is apparent from the case-law of the European Court of Human Rights that the relationship of a homosexual couple may fall within the notion of ‘private life’ and that of ‘family life’ in the same way as the relationship of a heterosexual couple in the same situation (ECtHR, 7 November 2013, Vallianatos and Othersv. Greece, CE:ECHR:2013:1107JUD002938109, § 73, and ECtHR, 14 December 2017, Orlandi and Othersv. Italy, CE:ECHR:2017:1214JUD002643112, § 143).
 
 51
-In the light of all the foregoing considerations, the answer to the first question is that, in a situation in which a Union citizen has made use of his freedom of movement by moving to and taking up genuine residence, in accordance with the conditions laid down in Article 7(1) of Directive 2004/38, in a Member State other than that of which he is a national, and, whilst there, has created or strengthened a family life with a third-country national of the same sex to whom he is joined by a marriage lawfully concluded in the host Member State, Article 21(1) TFEU 
+In the light of all the foregoing considerations, the answer to the first question is that, in a situation in which a Union citizen has made use of his freedom of movement by moving to and taking up genuine residence, in accordance with the conditions laid down in Article 7(1) of Directive 2004/38, in a Member State other than that of which he is a national, and, whilst there, has created or strengthened a family life with a third-country national of the same sex to whom he is joined by a marriage lawfully concluded in the host Member State, Article 21(1) TFEU must be interpreted as precluding the competent authorities of the Member State of which the Union citizen is a national from refusing to grant that third-country national a right of residence in the territory of that Member State on the ground that the law of that Member State does not recognise marriage between persons of the same sex.
```

Comment:
Member States are not allowed to not recognize same sex marriages -> same sex marriages must be recognized